python/lsst/ts/xml/data/sal_interfaces/MTReflector/MTReflector_Commands.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>MTReflector</Subsystem>
+    <EFDB_Topic>MTReflector_command_toggle</EFDB_Topic>
```

Comment:
Please, create an open and a close command instead of a `toggle` command.

Reply:
Please remove the toggle command.